docs/source/multiprocessing_example.rst

```diff
@@ -0,0 +1,92 @@
+Extracating Features in parallel using multiprocessing/scoop
```

Comment:
You don't have to add the rst files in docs/source, they are generated automatically from the notebooks. You can remove this file and extrafeats_example.rst